mainapp/models.py

```diff
@@ -165,6 +176,10 @@ class Contributor(models.Model):
     phone = models.CharField(max_length=10)
     address = models.TextField()
     commodities = models.TextField(verbose_name="What you can contribute. ( സംഭാവന ചെയ്യാന്‍ ഉദ്ദേശിക്കുന്ന സാധനങ്ങള്‍ ) -- Eg: Shirts, torches etc ")
+    blood_group=models.CharField(
+        max_length=5,
```

Comment:
@adityakuppa26 Shouldn't max length be 3? as you will not be having more than 3 chars for a blood group. Any specific reason this has been set to 5?